From 73e740b17d47e3ccddc72b58221f21a431b49d46 Mon Sep 17 00:00:00 2001
From: JP-Ellis <josh@jpellis.me>
Date: Mon, 2 Dec 2019 11:33:52 +1100
Subject: [PATCH] Remove upper limits on dependencies

Upper limits can be re-instated if/when an incompatibility is discovered (and
until papis can be updated to be compatible with the newer version).

Signed-off-by: JP-Ellis <josh@jpellis.me>
---
 setup.py | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/setup.py b/setup.py
index 38cfbe0c..7e7c3bf9 100644
--- setup.py
+++ setup.py
@@ -65,15 +65,17 @@
         "bibtexparser>=0.6.2",
         "click>=7.0.0",
         "habanero>=0.6.0",
-        "isbnlib>=3.9.1,<4.0.0",
-        "prompt_toolkit>=2.0.5,<3",
+        "isbnlib>=3.9.1",
+        "prompt_toolkit>=2.0.5",
         "tqdm>=4.1",
         "pygments>=2.2.0",
         "stevedore>=1.30",
         "python-doi>=0.1.1",
         # for python 3.4
-        "lxml<=4.3.5",
-        "python-slugify>=1.2.6,<4",
+        "lxml<=4.3.5 ; python_version<='3.5'",
+        "lxml>=4.3.5 ; python_version>'3.5'",
+        "python-slugify>=1.2.6,<4 ; python_version<='3.4'",
+        "python-slugify>=1.2.6 ; python_version>'3.4'",
     ],
     python_requires='>=3',
     classifiers=[
